File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int len = 0;
        bool endOfString = false;
        for(int i=s.length()-1; i>=0 ; i--){
            if(s[i] != ' ') endOfString = true;

            if(s[i] == ' ' && endOfString){
                break;
            }

            if(endOfString){
                len++;
            }
        }
        return len;
    }
};

File: 58-length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    s = s.trim();
    let lengthOfLastWord = 0;

    for(let i= s.length -1 ; (i >= 0 && s[i] !== ' ') ; i--){
        lengthOfLastWord++;
    }

    return lengthOfLastWord;
};